R/joyn-merge.R

```diff
@@ -292,10 +303,8 @@ joyn <- function(x,
     by, yvars_w
   )
 
-  if (
-    (keep == "anti" | keep == "semi") &
-    match_type == "m:m"
-  ) stop(
+  if (keep == "semi" &
+      match_type == "m:m") stop(
```

Comment:
Is this condition ok? I thought you had to remove `"semi"` rather than `"anti"`... Please disregard this comment is does not make sense.

Reply:
Hi @randrescastaneda , I believe what we are doing here works. However, even the condition of `keep == "semi"` is irrelevant because we do not yet allow semi joins for `joyn::joyn()`. I am going to remove this condition.

I did, however, add more tests for `joyn()` to test that the m:m anti join works since you identified this.